content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -129,17 +129,28 @@ Plugins wishing to perform "pre-reserve" work should use the
 NormalizeScore extension point.
 {{< /note >}}
 
-### Reserve
-
-This is an informational extension point. Plugins which maintain runtime state
-(aka "stateful plugins") should use this extension point to be notified by the
-scheduler when resources on a node are being reserved for a given Pod. This
-happens before the scheduler actually binds the Pod to the Node, and it exists
-to prevent race conditions while the scheduler waits for the bind to succeed.
-
-This is the last step in a scheduling cycle. Once a Pod is in the reserved
-state, it will either trigger [Unreserve](#unreserve) plugins (on failure) or
-[PostBind](#post-bind) plugins (on success) at the end of the binding cycle.
+### Reserve {#reserve}
+
+The Reserve extension point backs two informational scheduling phases called
+Reserve and Unreserve which are implemented in pairs and get called during these
```

Comment:
```suggestion
A plugin that implements the Reserve extension has two methods, namely `Reserve` and `Unreserve`
that back two informational scheduling phases, `Reserve` and `Unreserve`, respectively.
```

TBH, `implemented in pairs` sounds very confusing to me.

Reply:
Done.